phpBB/styles/prosilver/theme/content.css

```diff
@@ -466,17 +466,24 @@ blockquote cite {
 	/* Username/source of quoter */
 	font-style: normal;
 	font-weight: bold;
-	margin-left: 20px;
+	margin-left: 0;
 	display: block;
 	font-size: 0.9em;
 }
 
+blockquote cite:before, .uncited:before {
+	/* Font Awesome quote-left */
+	content: '\f10d';
+	font-family: FontAwesome;
+	padding-right: 5px;
+}
+
 blockquote cite cite {
 	font-size: 1em;
 }
 
 blockquote.uncited {
-	padding-top: 25px;
+	padding-top: 0;
 }
```

Comment:
as you should chrome's tool is the best way to work,
I do use it for testing but this was a guess based on the code. 
1. Since we are not applying an img and using `before` it gives us a new box model
2. if you are setting everything to zero this is a clue that it probably can be removed.